domain/relation/watcher_test.go

```diff
@@ -415,98 +415,101 @@ func (s *watcherSuite) TestWatchRelatedUnitsUnitScope(c *tc.C) {
 }
 
 // TODO: fix this test, currently failing with:
-// panic: test timed out after 10m0s
+//
+//	panic: test timed out after 10m0s
+//
```

Comment:
I don't know if it's a git or github problem but these lines should go away right?